app/assets/stylesheets/views/comments.scss

```
@@ -269,3 +270,8 @@
     color: var(--base-90);
   }
 }
+
+.personal-template-button:not(.active),
+.moderator-template-button:not(.active) {
+   @extend .crayons-btn--outlined;
+}
```

I'm going to add the new line via suggestion, to avoid having to ask you for that change :)

```suggestion
}

```